src/pages/Login/Login.js

```diff
@@ -35,24 +51,24 @@ const Login = () => {
             className="loginMainInputId"
             type="text"
             placeholder="이메일을 입력해주세요"
-            name="id"
-            value={id}
+            name="email"
+            value={email}
             onChange={handleInput}
           />
           <br />
           <input
             className="loginMainInputPw"
             type="password"
             placeholder="비밀번호를 입력해주세요"
-            name="pw"
-            value={pw}
+            name="password"
+            value={password}
             onChange={handleInput}
           />
         </div>
         <div>
           <button
             className={isValid ? 'buttonLogin' : 'buttonLoginDisabled'}
-            disabled={id === '' || pw === '' ? true : false}
+            disabled={email === '' || password === '' ? true : false}
```

Comment:
```suggestion
            disabled={email === '' || password === ''}
```
해당 조건의 반환값 자체가 boolean 값이기에 이렇게 넣으셔도 됩니다!